backend/promptfoo/create_esg_report_config.yaml

```diff
@@ -0,0 +1,110 @@
+description: "Test ESG Report Prompt"
+
+providers:
+  - id: mistral:mistral-large-latest
+    config:
+      temperature: 0
+
+prompts: file://promptfoo_test_runner.py:create_prompt
+
+tests:
+  - description: "Sample test to aid in ESG report generation development"
+    vars:
+      user_prompt_template: "create-esg-report-user-prompt"
+      system_prompt_template: "create-esg-report-system-prompt"
+      user_prompt_args:
+        document_text: "Published September 2024  Carbon Reduction Plan 
```

Comment:
I tried running your prompt with the kingfisher business report (https://scottlogic.atlassian.net/wiki/spaces/FS/pages/4422729729/Kingfisher+Responsible+Business+Report) it cut off the response after the 2nd bullet point and social and didn't even get to Governance. The response is probably too long.

Reply:
Also it completely doesn't work for the McDonald's impact report (https://scottlogic.atlassian.net/wiki/spaces/FS/pages/4422696964/McDonald+s+Impact+Report+2023+24)